modules/references/client/components/read-references/ReferenceCounts.js

```diff
@@ -2,6 +2,7 @@ import { useTranslation } from 'react-i18next';
 import PropTypes from 'prop-types';
 import React from 'react';
 import styled from 'styled-components';
+import { experienceType } from '../../../experiences.prop-types';
```

Comment:
This import was failing, and I changed it  to `@/modules/references/experiences.prop-types` locally to get it work.

I don't have strong opinion but slight preference to `@/...`  imports over multiple `../`s, just for clarity.